packages/web-components/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export * from './badge/index.js';
 export * from './counter-badge/index.js';
+export * from './divider/index.js';
 export * from './progress-bar/index.js';
 export * from './spinner/index.js';
 export * from './text/index.js';
-
```

Comment:
nit: the empty line was intentional to have the components separated from the other exports :-)